instrumentation/opentelemetry-instrumentation-aiokafka/tests/test_utils.py

```diff
@@ -120,6 +163,8 @@ async def wrap_send_helper(
         tracer = mock.MagicMock()
         produce_hook = mock.AsyncMock()
         original_send_callback = mock.AsyncMock()
+        original_send_callback.side_effect = original_send
+        original_send_callback.return_value = SEND_RETURN_VALUE
```

Comment:
Aren't these exclusive?

Reply:
original_send is used to mock the parameter signature of the original method being wrapped. The original mock method was unable to trigger this bug（because of any parameter can by ok）